ruby-padrino/manifest.yml

```diff
@@ -3,3 +3,5 @@ applications:
 - name: test-ruby-padrino
   command: bundle exec unicorn -p $PORT -c ./config/unicorn.rb 
   memory: 256M
+  random-route: true
```

Comment:
Why do we want this setting?

Reply:
Hey Mark, so in my case and in a few others it will be deployed in a different space and get mad at you for trying to deploy it in your current one. This stops route overlapping and makes deploying it easier without having to hunt. 

Most of our manifests use it for that purpose.
I was getting an error about deploying it in multiple spaces
"routes cannot be mapped to destinations in different spaces"